app/metro.config.js

```diff
@@ -9,7 +9,7 @@
  * React Native CLI configuration file
  */
 'use strict';
-
+// CR * what is metro?
```

Comment:
As someone who isn't very aware of React Native's ecosystem, it seems from the name and code location that `metro` may be a bit like `expo`. Have you been searching how to solve installation/build/dependency problems with `metro` in mind? 🚆